charts/zenko/values.yaml

```diff
@@ -80,6 +80,8 @@ prometheus:
     enabled: false
   pushgateway:
     enabled: false
+  server:
+    replicaCount: 2
```

Comment:
Does this work without any chart changes?

Reply:
No 😥